src/Http/Controllers/Api/v1/ApiKeyController.php

```diff
@@ -83,7 +83,14 @@ public function store(ApiKeyValidator $request)
     public function show($id)
     {
 
-        return ApiKey::with('info', 'characters')->findOrFail($id);
+        if (!is_numeric($id))
```

Comment:
We should rather use `$this->validate()` with a validator to manage the validation for us. This applies to all of the checks added. Could you update those?